doc/apimanager-reference.md

```diff
@@ -432,3 +450,163 @@ existing secret names.
 | username | In case the mail server requires authentication and the authentication type requires it | `""` |
 | password | In case the mail server requires authentication and the authentication type requires it | `""` |
 | openssl.verify.mode | When using TLS, you can set how OpenSSL checks the certificate. This is really useful if you need to validate a self-signed and/or a wildcard certificate. You can use the name of an OpenSSL verify constant: `none` or `peer` | `""` |
+
+
+### Default APIManager's components compute resources
+
+When APIManager's `spec.resourceRequirementsEnabled` attribute is set to
+`true`, default compute resources are set for the APIManager components.
+
+The specific compute resources default values that are set for the
+APIManager components are the following ones:
+
+#### system-app's system-master container
```

Comment:
Up to you to change or leave like this. What about table format? it could be more readable
| **Component** | **CPU request**| **CPU limit** | **Memory request** | **Memory request** |
| --- | --- | --- | --- | --- |
| system-master | 50m | 1000m | 600Mi | 800Mi |

Reply:
Table format looks better, more compact. Changed :+1: